includes/admin/views/html-admin-page-status-tools.php

```diff
@@ -20,11 +20,11 @@
 		</thead>
 		<tbody class="tools">
 			<?php foreach ( $tools as $action => $tool ) : ?>
-				<tr>
+				<tr class="<?php echo $action; ?>">
```

Comment:
This should be wrapped in [esc_attr](https://codex.wordpress.org/Function_Reference/esc_attr).